packages/hash/shared/src/ProsemirrorManager.ts

```diff
@@ -171,6 +176,16 @@ export class ProsemirrorManager {
         ? childrenForTextEntity(blockData, this.schema)
         : [];
 
+    /**
+     * Wrap the component node itself (rendered by ComponentView) in the
+     * following:
+     *
+     *    1. An entity node to store draft ids for the Text entity (if any)
+     *       linked to the block
+     *    2. An entity node to store ids for the entity linked to the block
+     *    3. [Outermost] The block node (rendered by BlockView) which
+     *       provides the surrounding UI
+     */
```

Comment:
This is no longer accurate as written, as the inner entity node is now *always* the block data entity's draft ID, instead of being overwritten by the inner text entity's draft ID if it exists as we did before. 

```suggestion
    /**
     * The structure of this is as follows:
     *
     * Block node (BlockView) to render the wrapping block UI, i.e, block handle
     * -> Entity node, which renders no UI, to store the block entity draft id
     *   -> Entity node, to store the block data entity's draft id
     *     -> The component node (ComponentView), to render the actual block component
     */
```